app/cafe.py

```diff
@@ -0,0 +1,25 @@
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+import datetime
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def __str__(self) -> str:
+        return self.name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError
+        expiration_date = visitor["vaccine"].get("expiration_date")
+        if expiration_date is None:
+            raise NotVaccinatedError
```

Comment:
It seems that this second check is unnecessary here, plus you have code duplication (the same `raise` twice). If you want to check this, it's better to do it like this
```suggestion
        if (
            visitor.get("vaccine") is None 
            or visitor["vaccine"].get("expiration_date") is None
        ):
            raise NotVaccinatedError
```